libsolidity/ast/ASTAnnotations.h

```diff
@@ -168,6 +168,8 @@ struct ContractDefinitionAnnotation: TypeDeclarationAnnotation, StructurallyDocu
 	/// List of contracts whose bytecode is referenced by this contract, e.g. through "new".
 	/// The Value represents the ast node that referenced the contract.
 	std::map<ContractDefinition const*, ASTNode const*, ASTCompareByID<ContractDefinition>> contractDependencies;
+
+	std::map<FunctionDefinition const*, util::SetOnce<uint64_t>> internalFunctionIDs;
```

Comment:
```suggestion
	std::map<FunctionDefinition const*, uint64_t> internalFunctionIDs;
```
We can probably simplify this and get rid of the ``SetOnce`` this way...